qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -547,6 +547,33 @@ def mine_quorum(self, expected_valid_count=10):
 
         sync_blocks(self.nodes)
 
+    def _start_logging(self):
```

Comment:
This block/function should be moved to line 210 to be a part of `BitcoinTestFramework` and not `DashTestFramework`.